rockcraft/extensions/extension.py

```diff
@@ -87,8 +87,8 @@ def validate(self, extension_name: str) -> None:
             "ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS"
         ):
             raise errors.ExtensionError(
-                f"Extension is experimental: {extension_name!r}",
-                docs_url="https://rockcraft.io/docs/supported-extensions",
+                f"Extension is experimental: {extension_name!r}, please run with ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS=True",
+                docs_url="https://documentation.ubuntu.com/rockcraft/en/latest/reference/extensions/",
```

Comment:
you can now refer to the slug and the correct version is shown depending on the version of rockcraft that prints the error, this is still correct and fixes the issue of referring to the deprecated link